library/src-bootstrapped/scala/quoted/util/ExprMap.scala

```diff
@@ -64,11 +64,7 @@ trait ExprMap {
           val tp = tpt.tpe match
             // TODO improve code
             case AppliedType(TypeRef(ThisType(TypeRef(NoPrefix(), "scala")), "<repeated>"), List(tp0: Type)) =>
-              // TODO rewrite without using quotes
-              type T
-              val qtp: quoted.Type[T] = tp0.seal.asInstanceOf[quoted.Type[T]]
-              given qtp.type = qtp
-              '[Seq[T]].unseal.tpe
+              Symbol.requiredClass("scala.collection.immutable.Seq").typeRef.appliedTo(tp0)
```

Comment:
It seems for this typical usage, an API like `def fromPath(path: String): Type` is more friendly:

```Scala
Type.fromPath("scala.collection.immutable.Seq").appliedTo(tp0)
```

The _compiler API_ can keep the same, the _user API_ can be a little different. This way, we also defend against compiler changes from meta-programmers.